gapis/api/api.go

```diff
@@ -46,7 +46,7 @@ type API interface {
 		after []uint64,
 		state *GlobalState,
 		thread uint64,
-		attachment FramebufferAttachment) (width, height, index uint32, format *image.Format, err error)
+		attachment FramebufferAttachment) (width, height, index uint32, format *image.Format, canResize bool, err error)
```

Comment:
There comes a point where a struct is preferred over N return values 😃. I suggest declaring a `FramebufferAttachmentInfo` struct in this file with `Width`, `Height`, `Index`, `Format` and `CanResize` fields.